[IP-988]: Template Layout Redesign

iEscapedVim opened the pull request:
## Description
Updated the layout of Invoice/Quote Templates. To a bit of a modern-looking layout.


## Related Issue
I couldn't find any latest to it.

## Motivation and Context
Well, the current layouts of invoice/quote templates are old and don't look good.

## Screenshots (if appropriate):
![image](https://github.com/InvoicePlane/InvoicePlane/assets/32537564/14145aae-584e-416d-bd6b-42746fbf10b4)

![image](https://github.com/InvoicePlane/InvoicePlane/assets/32537564/c984ca42-eb1b-4f8f-ab2b-bf5c5a4eaede)


## Pull Request Checklist

  * [x] My code follows the code formatting guidelines.
  * [x] #980 
  * [x] I selected the corresponding branch.
  * [x] I have rebased my changes on top of the corresponding branch.

## Issue Type (Please check one or more)

  * [ ] Bugfix
  * [x] Improvement of an existing Feature
  * [ ] New Feature


<!-- This is an auto-generated comment: release notes by coderabbit.ai -->

## Summary by CodeRabbit

- New Features
  - Overdue and Paid watermarks on PDFs.
  - Terms/Notes sections rendered when provided.
  - Inline “Pay Now” button on public invoices when online payments are enabled.
  - Dynamic document titles include the invoice/quote number.
- Enhancements
  - Complete redesign of invoice and quote templates (PDF and web) with clearer company/client sections and “Invoiced/Quoted To” headers.
  - Improved item tables with optional discounts, item taxes, per-rate taxes, and clearer totals (subtotal, total, paid, balance).
  - Better layout for dates, due/expire dates, amount due, and payment method.
- Style
  - New PDF/Web CSS utilities for tables, spacing, widths, and typography for improved readability.

<!-- end of auto-generated comment: release notes by coderabbit.ai -->

iEscapedVim committed:
Template

mPDF Templates Update

spacecabbie commented:
This more a template I think you can find some more at: https://github.com/spacecabbie/IP-templates

iEscapedVim committed:
remove custom sections

iEscapedVim committed:
Merge branch 'InvoicePlane:development' into mPDF_Templates

iEscapedVim committed:
Update InvoicePlane.php

missing notes section, added condition to discount, (only visible if discount given)

iEscapedVim committed:
Update InvoicePlane_Web.php

same changes as mPDF template

iEscapedVim committed:
Merge branch 'InvoicePlane:development' into mPDF_Templates

nielsdrost7 committed:
Apply suggestions from code review

Co-authored-by: coderabbitai[bot] <136622811+coderabbitai[bot]@users.noreply.github.com>

nielsdrost7 committed:
Update .github/workflows/README.md

Co-authored-by: coderabbitai[bot] <136622811+coderabbitai[bot]@users.noreply.github.com>

nielsdrost7 committed:
Delete .github/workflows/quickstart.yml

nielsdrost7 committed:
Merge branch 'prep/v170' into mPDF_Templates
